CMSgov/design-system
# [WNMGDS-1610] Add analytics-logging controls to Storybook

## Summary

For https://jira.cms.gov/browse/WNMGDS-1610, I wanted to have an easy way for people to test out analytics, so I've added a control menu to Storybook for turning analytics mocking on and off.

## How to test

1. `yarn storybook`
2. Go to the Dialog story
3. Click on the little bell icon in the toolbar and turn on analytics logging
    <img width="171" alt="Screen Shot 2022-04-29 at 2 56 15 PM" src="https://user-images.githubusercontent.com/7595652/166074171-785f18d4-0fc3-4f29-a750-b9169a87fb53.png">
4. Open and close the dialog, and see the events logged to the browser's JavaScript console
